### PR TITLE
Update series.year.py

### PR DESCRIPTION
**Pull request by brentomagic**

### CODE REVIEW

File: narwhals/series.py [hunk 1]
```diff
@@ -805,7 +805,7 @@ def year(self) -> Series:
             >>> func(s_pd)
             0    2012
             1    2023
-            dtype: int...
+            dtype: int32
```
Comment (MarcoGorelli) [inline narwhals/series.py:808]: I think `int...` was fine?

Comment (brentomagic) [reply narwhals/series.py:808]: ok..let me try it again

**Comment by MarcoGorelli** [comment]
this PR shows 0 files changed - not sure what this is meant to do, sorry 😄 was it opened by accident?

**Comment by MarcoGorelli** [comment]
yup!

**Pull request closed without merging**